<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="generator" content="pandoc" />
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <meta name="author" content="Jesús Vélez Santiago" />
    <meta name="date" content="2021-08-22" />
    <title>Jesús Vélez Santiago | Resume</title>

    
        <script src="index_files/header-attrs-2.10/header-attrs.js"></script>
        <link href="index_files/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
        <link href="index_files/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />
        <link href="index_files/paged-0.15/css/resume.css" rel="stylesheet" />
        <script src="index_files/paged-0.15/js/config.js"></script>
        <script src="index_files/paged-0.15/js/paged.js"></script>
        <script src="index_files/paged-0.15/js/hooks.js"></script>
    
        <link rel="stylesheet" href="override.css" type="text/css" />
    
  </head>
  <body>

<div id="aside" class="section level1">
<h1>Aside</h1>
<p><img src="https://avatars.githubusercontent.com/u/46458320?v=4" alt="Jesús Vélez Santiago" /></p>
<div id="contact" class="section level2">
<h2>Contact Info</h2>
<ul>
<li><i class="fa fa-globe"></i> <a href="https://jvelezmagic.com/">jvelezmagic.com</a></li>
<li><i class="fa fa-github"></i> <a href="https://github.com/jvelezmagic">@jvelezmagic</a></li>
<li><i class="fa fa-twitter"></i> <a href="https://twitter.com/jvelezmagic">@jvelezmagic</a></li>
<li><i class="fa fa-linkedin"></i> <a href="https://www.linkedin.com/in/jvelezmagic/">@jvelezmagic</a></li>
<li><i class="fa fa-envelope"></i> <a href="mailto:jvelezmagic@gmail.com" class="email">jvelezmagic@gmail.com</a></li>
</ul>
</div>
<div id="skills" class="section level2">
<h2>Skills</h2>
<ul>
<li><p>Experienced in machine learning, statistical and mathematical modeling,
data analysis, and data visualization.</p></li>
<li><p>Highly skilled in R, Python, Git, and Conda.</p></li>
</ul>
</div>
<div id="disclaimer" class="section level2">
<h2>Disclaimer</h2>
<p>Last updated on 2021-08-22.</p>
</div>
</div>
<div id="main" class="section level1">
<h1>Main</h1>
<div id="title" class="section level2">
<h2>Jesús Vélez Santiago</h2>
<div id="genomic-scientist-data-scientist" class="section level3">
<h3>Genomic Scientist | Data Scientist</h3>
<p>I love analyzing and visualizing data to improve our understanding of what is
around us. Besides, I am fascinated by software development, and I hope to
contribute valuable packages to the scientific community one day. I am known
for being a passionate, curious person and be always open to new challenges.
Do you have any challenges? Good! Let’s do it. Let’s be problem solvers.</p>
</div>
</div>
<div id="education-and-training" class="section level2" data-icon="graduation-cap" data-concise="true">
<h2>Education and Training</h2>
<div id="datacamp-inc." class="section level3">
<h3>Datacamp Inc.</h3>
<p>Data Scientist Professional</p>
<p><a href="https://www.datacamp.com/certificate/DS0014043984873">Certificate</a></p>
<p>2021</p>
<p>The following knowledge was assessed:</p>
<div class="concise">
<ul>
<li>Data Management</li>
<li>Model Development</li>
<li>Exploratory Analysis</li>
<li>Coding for Production Environments</li>
<li>Statistical Experimentation</li>
<li>Communication and Reporting</li>
</ul>
</div>
</div>
<div id="national-autonomous-university-of-mexico-unam" class="section level3">
<h3>National Autonomous University of Mexico (UNAM)</h3>
<p>B.S. in Genomic Sciences</p>
<p>Cuernavaca, Morelos, Mexico</p>
<p>2017 - 2021</p>
<p>Thesis: <a href="https://jvelezmagic.github.io/undergraduate_research_project/">Bacterial filamentation: a bet for survival in stressful environments</a></p>
</div>
</div>
<div id="research-experience" class="section level2" data-icon="laptop">
<h2>Research Experience</h2>
<div id="undergrad-thesis-student" class="section level3">
<h3>Undergrad Thesis Student</h3>
<p>Center for Genomic Sciences</p>
<p>Cuernavaca, Morelos, Mexico</p>
<p>2018 - 2021</p>
<ul>
<li>I combined mathematical models with experimental microbiology to understand
how exposure to antimicrobial substances triggers bacterial cells’ stress
responses. I have mainly focused on postulating a mathematical model
showing that cell elongation can promote transient resistance to individual
bacterial cells by diluting the antibiotic’s intracellular concentration.</li>
</ul>
</div>
<div id="developer-of-r-software" class="section level3">
<h3>Developer of R Software</h3>
<p>Universität Heidelberg</p>
<p>Remote work, Germany</p>
<p>2020 - 2021</p>
<ul>
<li>Over the years, many different regulatory networks and statistical algorithms
have been developed, mostly in a fixed combination of one network and one
algorithm. To systematically evaluate both networks and algorithms,
we developed <a href="https://saezlab.github.io/decoupleR/">decoupleR</a>, an R package
that allows users to apply efficiently any combination provided.</li>
</ul>
</div>
<div id="assitant-of-national-researcher-level-iii-or-emeritus" class="section level3">
<h3>Assitant of National Researcher Level III or Emeritus</h3>
<p>Center for Genomic Sciences</p>
<p>Cuernavaca, Morelos, Mexico</p>
<p>2019 - 2020</p>
<ul>
<li>Perform analysis of collections of massive expression experiments
(<em>e.g.</em>, RNA-seq) from <em>E. coli</em> using uniform bioinformatic pipelines
that allowed comparison with the transcriptional regulation network’s
classical experimental knowledge.</li>
</ul>
</div>
<div id="bioinformatician-and-genomist" class="section level3">
<h3>Bioinformatician and Genomist</h3>
<p>Centré de Regulació Genòmica</p>
<p>Barcelona, Spain</p>
<p>2019</p>
<ul>
<li>Differential expression analysis between the different conditions
(<em>e.g.</em>, deletions, overexpression, wild type) of transcriptional factors.</li>
<li>Comparison of the expression profiles of genes regulated by transcriptional
factors with the information available in <em>E. coli</em> gene regulation
databases (<em>e.g.</em>, RegulonDB).</li>
</ul>
</div>
<div id="visiting-scientist" class="section level3">
<h3>Visiting Scientist</h3>
<p>Center for Genomic Sciences</p>
<p>Cuernavaca, Morelos, Mexico</p>
<p>2018</p>
<ul>
<li>Design of R programs for the study of the topological properties of the E. coli transcriptional regulatory network.</li>
</ul>
</div>
</div>
<div id="professional-experience" class="section level2" data-icon="suitcase">
<h2>Professional Experience</h2>
<div id="data-science-professor" class="section level3">
<h3>Data Science Professor</h3>
<p><a href="https://platzi.com/">Platzi Inc.</a></p>
<p>Remote work</p>
<p>2021</p>
<div class="concise">
<ul>
<li>Complete design and planning of two data science courses.</li>
<li>Creation of exams and support materials.</li>
<li>Courses promotional writing.</li>
<li>Courses promotional live class.</li>
</ul>
</div>
</div>
</div>
<div id="teaching-experience" class="section level2" data-icon="chalkboard-teacher">
<h2>Teaching Experience</h2>
<div id="advanced-customization-of-environment-for-data-science" class="section level3">
<h3>Advanced Customization of Environment for Data Science</h3>
<p><a href="https://platzi.com/">Platzi Inc.</a></p>
<p><a href="https://platzi.com/clases/entorno-ciencia-datos/">Link to my course</a></p>
<p>2021</p>
<p>Learn how to customize a data science workbench based on Python, Conda,
and Jupyter Notebooks. With these tools, you will achieve harmony in your
work environment and increase your productivity as a data scientist.</p>
</div>
<div id="jupyter-notebooks-for-data-science" class="section level3">
<h3>Jupyter Notebooks for Data Science</h3>
<p><a href="https://platzi.com/">Platzi Inc.</a></p>
<p><a href="https://platzi.com/clases/jupyter-notebook/">Link to my course</a></p>
<p>2021</p>
<p>Learn about the different types of Jupyter notebook, the fundamental
work tool for any data scientist.</p>
</div>
<div id="teaching-assistant-of-analysis-of-transcriptomic-data-of-single-cell-scrna-seq-with-r-and-bioconductor.-couse" class="section level3">
<h3>Teaching Assistant of “Analysis of transcriptomic data of single cell (scRNA-seq) with R and Bioconductor.” Couse</h3>
<p><a href="http://congresos.nnb.unam.mx/EBM2021/analisis-de-datos-transcriptomicos-de-single-cell-con-r/">Bioinformatics Meeting in Mexico 2021</a></p>
<p><a href="https://comunidadbioinfo.github.io/cdsb2021_scRNAseq/">Couse material</a></p>
<p>2021</p>
<div class="concise">
<ul>
<li>Provide support to students.</li>
<li>Solve configurations of work environments.</li>
<li>Guide discussions and exercises in breakout rooms.</li>
</ul>
</div>
</div>
<div id="teaching-assitant-of-linear-algebra-course" class="section level3">
<h3>Teaching Assitant of Linear Algebra Course</h3>
<p>Center for Genomic Sciences</p>
<p>Cuernavaca, Morelos, Mexico</p>
<p>2020</p>
<ul>
<li>Monitoring of student performance accompanied by personalized
and group tutorials.</li>
<li>Review of assignments and exams.</li>
</ul>
<p><br>
<br></p>
</div>
</div>
<div id="honors-and-awards" class="section level2" data-icon="medal">
<h2>Honors and Awards</h2>
<div id="winner-of-first-olimpiadas-platzi-data-triathlon-platzi" class="section level3">
<h3>Winner of first “Olimpiadas Platzi: Data Triathlon” – Platzi</h3>
<p><a href="https://platzi.com/">Platzi Inc.</a></p>
<p><a href="https://platzi.com/blog/premiacion-data-triathlon/">Award link</a></p>
<p>2020</p>
</div>
<div id="support-program-for-research-and-technological-innovation-projects-papiit" class="section level3">
<h3>Support Program for Research and Technological Innovation Projects (PAPIIT)</h3>
<p>National Autonomous University of Mexico (UNAM)</p>
<p>Mexico</p>
<p>2020 - 2021</p>
</div>
<div id="program-for-special-inter-institutional-cooperation-activities-paeci" class="section level3">
<h3>Program for Special Inter-institutional Cooperation Activities (PAECI)</h3>
<p>National Autonomous University of Mexico (UNAM)</p>
<p>Mexico</p>
<p>2019</p>
</div>
</div>
<div id="selected-publications-and-posters" class="section level2" data-icon="file">
<h2>Selected Publications and Posters</h2>
<div id="filamentation-a-bet-for-survival." class="section level3">
<h3>Filamentation: a bet for survival.</h3>
<p>Poster for 2019 Fall School of Mathematical Biology (EOBM).</p>
<p>Merida, Yucatan, Mexico</p>
<p>2019</p>
<p><strong>Velez-Santiago J</strong>, Peña-miller R.</p>
</div>
</div>
</div>

<script>
(function() {
  var i, s, j, el, els;
  var create_el = function(type, className, content, isHTML) {
    var el = document.createElement(type);
    if (className) el.className = className;
    if (content) {
      if (isHTML) {el.innerHTML = content;} else {el.innerText = content;}
    }
    return el;
  }

  // replace h2 title with h1
  var title = document.getElementById('title').querySelector('h2');
  title.parentNode.replaceChild(create_el('h1', false, title.innerHTML, true), title);

  // add the class .aside to #aside
  el = document.getElementById('aside');
  if (el) el.className += ' aside';

  // tweak class names of sections, and add icons
  els = document.getElementById('main').querySelectorAll('.level2');
  for (i = 0; i < els.length; i++) {
    el = els[i];
    if (i === 0 && el.id === 'title') continue;
    el.className += ' main-block';
    if (el.dataset['concise']) el.className += ' concise';
    // if there is no icon, add an icon:
    if (el.firstElementChild.firstChild && el.firstElementChild.firstChild.nodeName !== 'I') {
      s = el.dataset['icon'] || 'bookmark';
      el.firstElementChild.insertBefore(create_el('i', 'fa fa-' + s), el.firstElementChild.firstChild);
    }
  }

  // tweak class names of blocks in sections
  els = document.getElementById('main').querySelectorAll('.level3');
  for (i = 0; i < els.length; i++) {
    el = els[i];
    if (el.parentNode.id === 'title') continue;
    el.className += ' blocks';
    el.innerHTML = '<div class="date"></div>' + '<div class="decorator"></div>' +
      '<div class="details"><header></header><div></div></div>' + el.innerHTML;
    var ps = el.querySelectorAll('p');
    // move the date paragraph to the date div
    if (ps.length >= 3) {
      s = ps[2].innerText;
      s = s === 'N/A' ? [] : s.split(' - ');
      el.removeChild(ps[2]);
      for (j = 0; j < s.length && j < 2; j++) {
        el.children[0].appendChild(create_el('span', false, s[j]))
      }
    }
    // move title, description, location to the details div
    (function(h) {
      h.appendChild(el.children[3]);
      var p = el.children[3];  // description
      if (p.innerText !== 'N/A') {
        h.appendChild(create_el('span', 'place', p.innerHTML, true));
      }
      el.removeChild(p);
      p = el.children[3];  // location
      if (p.innerText !== 'N/A') {
        s = create_el('span', 'location', p.innerHTML, true);
        s.innerHTML = '<i class="fa fa-map-marker-alt"></i> ' + s.innerHTML;
        h.appendChild(s);
      }
      el.removeChild(p);
    })(el.children[2].firstElementChild);
    // move the rest of content in a block into the last div of the details div
    s = el.children[2];
    while (j = s.nextSibling) {
      if (j.className === 'aside') { s = j; continue; }
      el.children[2].lastElementChild.append(j);
    }
  }
})();
</script>


  </body>
</html>
